api/api.go

```diff
@@ -31,7 +31,7 @@ type FilterFlexAPI interface {
 
 // OutputQueue - An interface used to queue filter outputs
 type OutputQueue interface {
-	Queue(output *models.Filter) error
+	Queue(ctx context.Context, output *models.Filter) error
```

Comment:
Technically this is a breaking change to [a publicly released API](https://pkg.go.dev/github.com/ONSdigital/dp-filter-api@v0.0.0-20230731091332-87f5b26600c3/api#OutputQueue) consider extending the module instead if you can or else releasing as a V2 of the module (we use a branching v2 strategy rather than a dir based one if you do this).

Reply:
Ah ok, so you mean release a new major version of the dp-filter-api? Looks like we are currently on v2.29.0 so can release v3.0.0